text/0000-crypto.md

```diff
@@ -0,0 +1,62 @@
+- Feature Name: crypto
+- Start Date: 2016-05-22
+- RFC PR: (leave this empty)
+- Pony Issue: (leave this empty)
+
+# Summary
+
+The addition of a Cryptography package to the stdlib would provide necessary features to Pony applications, such as secure communications and password protection. The package will include a variety of common cryptographic algorithms such as SHA-1, SHA-256, MD5, DES, RSA, etc.
+
+# Motivation
+
+Cryptography is a vital tool in the areas of authentication, confidentiality, data integrity, and nonrepudiation.
+
+# Detailed design
+
+*Security must be the first priority of this package.*
+
+This package will use the OpenSSL crypto library through the C FFI. The following API will be used:
+
+- HashFunc
```

Comment:
I think the name `HashFn` would be more idiomatic with some of the types we already have in the standard library.